vars/gradleCheckFlakyTestDetector.groovy

```diff
@@ -49,7 +51,7 @@ void call(Map args = [:]) {
                         repoUrl: "https://github.com/opensearch-project/OpenSearch",
                         issueTitle: "[AUTOCUT] Gradle Check Flaky Test Report for ${failedTest}",
                         issueBodyFile: "${failedTest}.md",
-                        label: 'autocut,>test-failure',
+                        label: 'autocut,>test-failure,flaky-test',
```

Comment:
Should we take this as an arg? In case the labels change in future? Don't want to keep updating the library.

Reply:
Initially I had the same thought in my mind, but I dont think so this will need more labels moving forward, once created with default labels its upto repo maintainers to add more, I have seen the `flaky-test` on very old issues https://github.com/opensearch-project/OpenSearch/issues/1561 and hence it looks to me like standard label. WDYT @gaiksaya ?

Reply:
I still feel it should be arg. Another use case, if GH cli changes in future with `>` giving issues, we can easily switch update the label and change in the jenkinsFile. Take all the labels as arg no worries but its just easy manipulation.

Reply:
Sure let me make that change.

Reply:
@gaiksaya made this change can you please check ?

Reply:
Just tested an it works https://github.com/prudhvigodithi/OpenSearch/issues/27.

Reply:
Thanks! Looks good.